docs/graphql/manual/guides/pg-dump.rst

```diff
@@ -0,0 +1,26 @@
+Backup and Restore Postgres data running on a Docker Container
+===============================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 0
+  :local:
+
+Postgres comes with handy tools such as **pg_dump** and **psql** which lets us take backups and restore them with ease.
+
+To backup data, we use the pg_dump tool:
+
+.. code-block:: bash
+
+    docker exec <postgres_container_name> pg_dump -U postgres <database_name> > backup.sql
+
+.. note::
```

Comment:
Isn't this true for both commands? If yes, this note should be at the end of this section.

Reply:
Yes, I will add it to the end of the section